pow-migration/src/monitor.rs

```diff
@@ -72,7 +74,7 @@ pub fn generate_online_tx(validator: String) -> OutgoingTransaction {
         to: Address::burn_address().to_user_friendly_address(),
         value: 1, //Lunas
         fee: 0,
-        data: Some(hex::encode("online")),
+        data: Some(hex::encode(String::from(ONLINE_TX_DATA))),
```

Comment:
There is a comment above that could be updated, but it is not of great concern.

Reply:
Done.